aggregator/src/binaries/aggregator.rs

```diff
@@ -342,6 +343,12 @@ pub struct Config {
     /// specify this.
     #[serde(default)]
     pub global_hpke_configs_refresh_interval: Option<u64>,
+
+    /// Defines how long to cache tasks for, in seconds. This affects how often the aggregator
+    /// becomes aware of task parameter changes. If unspecified, default is defined by
+    /// [`TASK_AGGREGATOR_CACHE_DEFAULT_TTL`]. You shouldn't normally have to specify this.
+    #[serde(default)]
+    pub task_cache_ttl_seconds: Option<u64>,
```

Comment:
Similar to `global_hpke_configs_refresh_interval`, I haven't made note of this parameter. I can't think of a reason why somebody would actually need to change this, but I plumbed it through the config just in case.